src/components/CreatePostForm.tsx

```diff
@@ -0,0 +1,65 @@
+// Render Prop
+import React from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { PostItemInterface } from '../interfaces/Post';
+
+
+const CreatePostForm = ({
+  onSavePost
+}: { onSavePost: (i: PostItemInterface) => void}) => (
+  <div>
+    <h1>New Post</h1>
+    <Formik
```

Comment:
How would you go about testing this component?

Reply:
this form is easy, just check all the fields are ther and check the event on the button